<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelPass_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUser_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="regLineEditPassword">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="regLineEditUser">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPass_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Re-Type Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="regLineEditPasswordRetype">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelErrorUser">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;* Already Existing&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRetypePassError">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;* Not Matching&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxMonth">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>January</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Febuary</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>March</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>April</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>May</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>June</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>July</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>August</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>September</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>October</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>November</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>December</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelUser_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Month&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUser_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Year&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUser_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Day&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Date of Birth&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDay">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditYear">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelErrorDate">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;* Age is Less than 12&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxGender">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Gender</string>
   </property>
   <widget class="QRadioButton" name="radioButtonMale">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonFemale">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxAccounType">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Account Type</string>
   </property>
   <widget class="QRadioButton" name="radioButtonUser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonAdmin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>431</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Favourite Genre(s)</string>
   </property>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Comedy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Drama</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Horror</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Romance</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="regPushButtonRegister">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>510</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelErrorFull">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>520</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;* All Fields Must be Filled&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
